pkg/okteto/errors.go

```diff
@@ -0,0 +1,23 @@
+package okteto
+
+import "errors"
+
+var (
+	//ErrGithubMissingBusinessEmail is raised when the user does not have a business email
+	ErrGithubMissingBusinessEmail = errors.New("github-missing-business-email")
+)
+
+// IsErrGithubMissingBusinessEmail returns true if the error is ErrGithubMissingBusinessEmail
+func IsErrGithubMissingBusinessEmail(err error) bool {
+	return err.Error() == ErrGithubMissingBusinessEmail.Error()
```

Comment:
Check out `error.Is()`, maybe it's more suitable to check if the error is wrapped